client/packages/system/src/Encounter/DetailView/DetailView.tsx

```diff
@@ -63,6 +65,11 @@ export const DetailView: FC = () => {
           message={t('messages.click-to-return-to-encounters')}
         />
       )}
+      <Footer
+        onSave={saveData}
```

Comment:
Would be good if the save button also saves changes from the app bar fields, will have a look at it